open/text/embeddings/server/__main__.py

```diff
@@ -17,9 +17,10 @@
 """
 import uvicorn
 import os
-from open.text.embeddings.server.app import create_app
+from open.text.embeddings.server.app import create_app, initialize_embeddings
 
 if __name__ == "__main__":
+    initialize_embeddings()
```

Comment:
I will move the `initialize_embeddings()` to `create_app()` as the current approach will break the code of aws.py.